src/Media.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import elementType from 'react-prop-types/lib/elementType';
+import classNames from 'classnames';
+
+import Heading from './MediaHeading';
+import Body from './MediaBody';
+import Left from './MediaLeft';
+import Right from './MediaRight';
+import List from './MediaList';
+import ListItem from './MediaListItem';
+
+const Media = React.createClass({
+  displayName: 'Media.Media',
+  propTypes: {
+    /**
+     * You can use a custom element for the media container
+     */
+    componentClass: elementType
+  },
+
+  getDefaultProps() {
+    return {
+      componentClass: 'div'
+    };
+  },
+
+  render() {
+    const ComponentClass = this.props.componentClass;
```

Comment:
You can simplify this whole block as:

```js
const { ComponentClass, className, ...props } = this.props;

return (
  <ComponentClass {...props} className={classNames(className, 'media')} />
);
```

`children` is implicitly spread through as a prop.